charts/spire/charts/spiffe-oidc-discovery-provider/templates/configmap.yaml

```diff
@@ -23,9 +23,12 @@ domains:
   - "{{ include "spiffe-oidc-discovery-provider.fullname" . }}"
   - "{{ include "spiffe-oidc-discovery-provider.fullname" . }}.{{ include "spiffe-oidc-discovery-provider.namespace" . }}"
   - "{{ include "spiffe-oidc-discovery-provider.fullname" . }}.{{ include "spiffe-oidc-discovery-provider.namespace" . }}.svc.{{ include "spire-lib.cluster-domain" . }}"
-  {{- $uri := urlParse (include "spire-lib.jwt-issuer" .) }}
-  {{- $jwtIssuer := (default $uri.path $uri.host) }}
-  {{- uniq (concat (list $jwtIssuer) .Values.config.additionalDomains) | toYaml | nindent 2 }}
+  {{- $jwtDomain := .Values.config.jwtDomain }}
+  {{- if not $jwtDomain }}
+    {{- $uri := urlParse (include "spire-lib.jwt-issuer" .) }}
+    {{- $jwtDomain := (default $uri.path $uri.host) }}
```

Comment:
```suggestion
  {{- if eq $jwtDomain "" }}
    {{- $uri := urlParse (include "spire-lib.jwt-issuer" .) }}
    {{- $jwtDomain = (default $uri.path $uri.host) }}
```

Reply:
Empty string is "falsy", ref. https://helm.sh/docs/chart_template_guide/control_structures/#ifelse. Thanks for the var fix suggestion! It's crazy that Helm (or Go-template) allows me to redefine/shadow an existing variable. 😅